src/styles/layout.sss

```diff
@@ -41,19 +41,16 @@
 .main
     flex: 5 0 60%
     display: flex
-    box-sizing: border-box
+    overflow: scroll
```

Comment:
Why you need scroll for main window?
Scrolling should be inside panes themself

Reply:
Scrolling should be inside pane, only in horizontal layout, not in vertical layout. Vertically, the whole content should expand the pane.

Reply:
Then it should be under `@media screen and (max-width: 960px)` below.
But overflow should be `auto` instead of `scroll.
And most likely we'd need to use overflow-y instead of just overflow.